src/azure-cli/azure/cli/command_modules/redis/tests/latest/test_redis_scenario.py

```diff
@@ -309,6 +309,10 @@ def test_redis_cache_managed_identity(self, resource_group):
             self.check('length(identity.userAssignedIdentities)', 1)
         ])
 
+        self.cmd('az redis update -n {name} -g {rg} --set "publicNetworkAccess=Disabled"')
```

Comment:
Don't you need to re-record the yaml file after updating test?

Reply:
Please don't merge this yet. Yeah, this is pending.